### PR TITLE
Fix getSessions output

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
https://github.com/SeleniumHQ/selenium/wiki/JsonWireProtocol requires `/sessions` call to return a list of active session ids along with their capabilities.

### CODE REVIEW

**Comment by mykola-mokhnach** [comment]
@KazuCocoa FYI

File: lib/espresso-runner.js [hunk 1]
```diff
@@ -117,7 +120,37 @@ class EspressoRunner {
...
+        timeout: 500,
+        json: true,
+      });
+      const activeSessionIds = value.map((sess) => sess.id);
+      if (activeSessionIds.length) {
+        logger.debug(`The following obsolete sessions are still running: ${JSON.stringify(activeSessionIds)}`);
+        logger.debug('Cleaning up the obsolete sessions');
+        await B.all(activeSessionIds.map((id) =>
+          request.delete({
+            url: `http://${this.host}:${this.systemPort}/wd/hub/session/${id}`,
```
Comment (KazuCocoa) [inline lib/espresso-runner.js]: `http://${this.host}:${this.systemPort}/session/${id}`?

Comment (mykola-mokhnach) [reply lib/espresso-runner.js]: fixed

File: lib/espresso-runner.js [hunk 2]
```diff
@@ -117,7 +120,37 @@ class EspressoRunner {
     logger.info(`Repackaged espresso server ready: '${this.modServerPath}'`);
   }
 
+  async cleanupSessionLeftovers () {
+    logger.debug('Performing cleanup of automation leftovers');
+
+    try {
+      const {value} = await request.get({
+        url: `http://${this.host}:${this.systemPort}/wd/hub/sessions`,
```
[the remark this answers is not in the document]
Comment (mykola-mokhnach) [reply lib/espresso-runner.js]: fixed

**Comment by mykola-mokhnach** [comment]
Thanks for your help @KazuCocoa

File: espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/KReflectionUtils.kt [hunk 3]
```diff
@@ -96,4 +98,13 @@ object KReflectionUtils {
                     "'${instance::class.qualifiedName}'. Reason: ${e.message}'");
         }
     }
+
+    fun extractDeclaredProperties (instance: Any): Map<String, Any?> {
+        return instance::class.declaredMemberProperties
+                .fold(mutableMapOf<String, Any?>()) { acc, prop ->
+                    @Suppress("UNCHECKED_CAST")
+                    acc[prop.name] = (prop as KProperty1<Any, Any?>).get(instance)
```
Comment (KazuCocoa) [inline espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/KReflectionUtils.kt]: 👍

**Pull request merged**